src/index.html

```diff
@@ -1,17 +1,174 @@
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field-set">
+        <legend>
+          Registration
+        </legend>
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            maxlength="10" 
+            minlength="3" 
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="form-field">
+          Do you love cats?
+          <label class="form-radio">
+            <input 
+              type="radio" 
+              name="loveCats" 
+              autocomplete="off"
+            >
+            Yes
+          </label>
+          <label class="form-radio">
+            <input 
+              type="radio" 
+              name="loveCats" 
+              autocomplete="off"
+            >
+            Not
+          </label>
+        </div>
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="color" 
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="time" 
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select 
+            name="brandOfcars" 
+            required 
+            multiple 
+            autocomplete="off"
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-field">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="rateOurWork" 
+            value="0" 
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>
+          Additional info:
+        </legend>
+        <label class="form-field">
+          Comments:
+          <textarea name="feedback" autocomplete="off"></textarea>
+        </label>
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend" autocomplete="off">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+      <button type="submit">Submit</button>
+    </form>
   </body>
+
 </html>
```

Comment:
```suggestion
  </body>
</html>
```
Don't leave newlines between parent and child elements